boards/particle-argon/doc.txt

```diff
@@ -0,0 +1,24 @@
+/**
+@defgroup    boards_particle-argon Particle Argon
+@ingroup     boards
+@brief       Support for the Particle Argon
+
+### General information
+
+[Particle Argon](https://docs.particle.io/argon/) is a mesh-ready development kit
+that provides access to multiple communication protocols: BLE, 802.15.4 and WiFi.
+
+img src="https://docs.particle.io/assets/images/argon/argon-top.png"
+     alt="pinout" style="height:300px;"/>
+
+### Block diagrams and datasheets
+
+[Datasheet](https://docs.particle.io/assets/pdfs/datasheets/argon-datasheet.pdf)
```

Comment:
For all boards this link looks weird in the generated doxygen page, it's just floating around next to the Image. Could you improve this? Something similar to what is done for nrf52dk or how it is done for nucleo boards?

Reply:
I see what you mean. I'm not expert in doxygen rendering: I tried to reword and to add link differently, under the block diagram.